### PR TITLE
Commands: Added the test262 command

### PR DESCRIPTION
**Pull request by networkException**
This command fetches and outputs the results of the ecma262 tests
run in LibJS from https://libjs.dev/test262.

If no argument is given, the latest commit will be used. Otherwise
the first argument will be used to resolve a commit by hash.

If the first argument is "labels", an overview over the icons
used in the result output will be displayed.

### CODE REVIEW

File: src/apis/githubAPI.ts [hunk 1]
```diff
@@ -103,7 +103,8 @@ class GithubAPI {
     }
 
     async fetchSerenityManpageByUrl(url: string): Promise<ManPage | undefined> {
-        const pattern = /https:\/\/github\.com\/([\w/]*)\/blob\/master\/([\w/]*)\/man(\d)\/([\w/]*)\.md/;
+        const pattern =
+            /https:\/\/github\.com\/([\w/]*)\/blob\/master\/([\w/]*)\/man(\d)\/([\w/]*)\.md/;
```
Comment (IdanHo) [inline src/apis/githubAPI.ts]: This change is made in the second commit and then reverted in the third, can we just not touch it in both?

File: src/commandHandler.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import { Message } from "discord.js";
 import { IssueCommand, ManCommand, PlanCommand, PRCommand, QuickLinksCommand } from "./commands";
 import Command from "./commands/commandInterface";
+import { Test262Command } from "./commands/test262Command";
```
Comment (IdanHo) [inline src/commandHandler.ts:10]: this should be exported from `commands/index.ts` and then imported here with the rest of the commands

**Review by IdanHo — CHANGES_REQUESTED** [review]
A couple of things :)

**Review by IdanHo — APPROVED** [review]
LGTM, @linusg are you fine with this as well? (Screenshot of the current view:)
image

**Comment by linusg** [comment]
Absolutely, this is great! :tada:

**Comment by IdanHo** [comment]
Great, lets merge this then :)

**Pull request merged**